examples/opf/clients/cpu/model_params.py

```diff
@@ -202,7 +202,7 @@
 
         'clParams': {
             # Classifier implementation selection.
-            'implementation': 'cpp',
+            'implementation': 'py',
```

Comment:
Ok I see that it is because the `regionName` was changed to `SDRClassifierRegion`. Can you just change that back to CLAClassifierRegion for now since the SDRClassifier still has some quirks to work out and the C++ impl isn't ready yet?